tests/FINDOLOGIC/Export/Tests/DataHelperTest.php

```diff
@@ -107,10 +106,46 @@ public function numericValueProvider()
      *
      * @expectedException \FINDOLOGIC\Export\Helpers\AttributeValueLengthException
      */
-    public function testCharacterLimitCausesException()
+    public function testAttributeValueCharacterLimitCausesException()
     {
         $value = implode('', array_fill(0, 16384, '©'));
 
         DataHelper::checkAttributeValueNotExceedingCharacterLimit('some attribute', $value);
     }
+
+    /**
+     * Test if item id character limit of data helper causes exception when called outside item class.
+     *
+     * @expectedException \FINDOLOGIC\Export\Helpers\ItemIdLengthException
+     */
+    public function testItemIdCharacterLimitCausesException()
+    {
+        $id = implode('', array_fill(0, 256, 'a'));
+
+        DataHelper::checkItemIdNotExceedingCharacterLimit($id);
+    }
+
+    /**
+     * Test if group name character limit of data helper causes exception when called outside item class.
+     *
+     * @expectedException \FINDOLOGIC\Export\Helpers\GroupNameLengthException
```

Comment:
The `@expectedException` annotation is deprecated in more recent versions of PHPUnit, and I believe even removed in the most recent one. Consider using something like `$this->expectException(GroupNameLengthException::class);` instead.

Reply:
Done!